frontend/src/components/cards/PackageCard/types.ts

```diff
@@ -4,5 +4,4 @@ export type PackageCardProps = {
   icon: IconProps['icon'];
   title: string;
   summary: string;
-  pageRoute: string;
```

Comment:
I think you'll also need to update the component so it doesnt try to use this prop

Reply:
I did :)